### ISSUE TITLE
The checkpoint pretrained  on KIT-ML dataset

### ISSUE THREAD

**Issue by qinghuannn** [exitudio/BAMM#9]
Hi. Thanks for your nice work!
Could you release the checkpoint pretrained  on KIT-ML dataset?

**Comment by vinit-katariya** [comment]
@exitudio thank you for your great work! Would it be possible for you to release the checkpoint pre-trained on the KIT-ML dataset?

**Comment by exitudio** [comment]
Hi,
 You can download the [pretrain KIT](https://drive.google.com/file/d/1aZTEcZQgryx4Y1Gy40anT4K0TVRJhlnZ/view?usp=drive_link). I haven't tested it, so there may be some bugs in KIT. You can investigate the saved source code in ```./source``` 

P.S. For evaluation, the masking strategy is different from HumanML3D—it keeps the highest probability. Replacing ```./models/mask_transformer/transformer.py``` with ```./source/models/mask_transformer/transformer__8_res_2iter_noRepredAll_full.py``` should slightly improve the score.
